### PR TITLE
UCDOC 3067 - Add WhiteSource plugin docs

### PR DESCRIPTION
**Pull request by Naveennkg**

### CODE REVIEW

File: docs/UCV/ucv-ext-whitesource/downloads.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+TestNG - Downloads
```
Comment (osmanburucu) [inline docs/UCV/ucv-ext-whitesource/downloads.md]: Shouldn't this be Whitesource instead of TestNG?

**Review by osmanburucu — CHANGES_REQUESTED** [review]
* the downloads.md contains TestNG instead of Whitesource
* please remove the README.md.bak

File: docs/UCV/ucv-ext-whitesource/downloads.md (hunk 1 above)
Comment (Naveennkg) [reply docs/UCV/ucv-ext-whitesource/downloads.md]: Yes, it must be WhiteSource.

**Review by osmanburucu — APPROVED** [review]
Thank You!

**Pull request merged**